api/language-extensions/language-server-extension-guide.md

```diff
@@ -379,13 +395,13 @@ async function validateTextDocument(textDocument: TextDocument): Promise<void> {
     diagnostics.push(diagnostic);
   }
 
-  // Send the computed diagnostics to VS Code.
+  // Send the computed diagnostics to VSCode.
   connection.sendDiagnostics({ uri: textDocument.uri, diagnostics });
 }
 
 connection.onDidChangeWatchedFiles(_change => {
-  // Monitored files have change in VS Code
-  connection.console.log('We received a file change event');
+  // Monitored files have change in VSCode
```

Comment:
"VS Code" with a space.
We should also update the sample to use the correct grammar "a file change event".